cypher/cypher-docs/src/docs/dev/ql/administration/indexes-for-search-performance/index.asciidoc

```diff
@@ -110,37 +110,61 @@ Index provider and configuration can be specified using the `OPTIONS` clause.
 CREATE LOOKUP INDEX [index_name] [IF NOT EXISTS]
 FOR (n)
 ON EACH labels(n)
+[OPTIONS "{" option: value[, ...] "}"]
 ----
 | Create a node label lookup index.
 
-Token lookup indexes do not support any `OPTIONS` values.
+Index provider can be specified using the `OPTIONS` clause.
 
 | [source, cypher, role=noplay]
 ----
 CREATE LOOKUP INDEX [index_name] [IF NOT EXISTS]
 FOR ()-"["r"]"-()
 ON [EACH] type(r)
+[OPTIONS "{" option: value[, ...] "}"]
 ----
 | Create a relationship type lookup index.
 
-Token lookup indexes do not support any `OPTIONS` values.
+Index provider can be specified using the `OPTIONS` clause.
+
+| [source, cypher, role=noplay]
+----
+CREATE TEXT INDEX [index_name] [IF NOT EXISTS]
+FOR (n:LabelName)
+ON (n.propertyName)
+[OPTIONS "{" option: value[, ...] "}"]
+----
+| Create a text index on nodes.
+
+Index provider can be specified using the `OPTIONS` clause.
+
+| [source, cypher, role=noplay]
+----
+CREATE TEXT INDEX [index_name] [IF NOT EXISTS]
+FOR ()-"["r:TYPE_NAME"]"-()
+ON (r.propertyName)
+[OPTIONS "{" option: value[, ...] "}"]
+----
+| Create a text index on relationships.
+
+Index provider can be specified using the `OPTIONS` clause.
 
 | [source, cypher, role=noplay]
 ----
 DROP INDEX index_name [IF EXISTS]
 ----
-| Drop an index, either a b-tree, fulltext, or token lookup index.
+| Drop an index of any index type.
 | The command is optionally idempotent, with the default behavior to throw an error if you attempt to drop the same index twice.
 With `IF EXISTS`, no error is thrown and nothing happens should the index not exist.
 
 | [source, cypher, role=noplay]
 ----
-SHOW [ALL\|BTREE\|FULLTEXT\|LOOKUP] INDEX[ES]
+SHOW [ALL\|BTREE\|FULLTEXT\|TEXT\|LOOKUP] INDEX[ES]
```

Comment:
If want them alphabetically, LOOKUP should be before TEXT.

Reply:
I'm planning on adding RANGE between BTREE and FULLTEXT... :P (since range is the common replacement for btree) and maybe text should be before fulltext as well though

Or I guess I could update to have all in alphabetical order

Reply:
I think either a logical or alphabetical order is ok, this is not really any of it :P

Reply:
might just be easier to add it in alphabetical, then it takes less thinking to update :P